flink-batch-connectors/flink-hbase/src/test/resources/log4j-test.properties

```diff
@@ -15,9 +15,16 @@
 # specific language governing permissions and limitations
 # under the License.
 
-log4j.rootLogger=${hadoop.root.logger}
-hadoop.root.logger=INFO,console
-log4j.appender.console=org.apache.log4j.ConsoleAppender
-log4j.appender.console.target=System.err
-log4j.appender.console.layout=org.apache.log4j.PatternLayout
-log4j.appender.console.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss} %p %c{2}: %m%n
+log4j.rootLogger=DEBUG, stdout, file
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.threshold=INFO
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{ABSOLUTE} %-5p %30c{1}:%4L - %m%n
+## file appender
```

Comment:
Can you remove the file appender configuration? This creates a 3.5 MB file and make the test heavier. We are suffering from long build times and try to keep new tests as lightweight as possible. Thanks